modules/terms-and-conditions/backend/modules/migrations/0001_initial.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.19 on 2021-04-26 18:42
```

Comment:
@danielsousaio I don't believe I did, this was generated for me with this name if I recall correctly. Should I rename it, or rerun it to try for a different name?

Reply:
Yes, it should be named uniquely, such as this example:
https://github.com/crowdbotics/modules/tree/develop/modules/django/articles/migrations

If you prefer you can also fingerprint the filename with its md5 checksum, akin to what Rails does. We need to create an helper utility to generate migration files names for module authors, this will be problematic otherwise.